app/controllers/assignments_controller.rb

```diff
@@ -33,12 +36,25 @@ def index
   end
 
   def new
-    @assignment = Assignment.new
-    @assignment.course_id = @course.id
-    @assignment.due_date = (Time.now + 1.week).end_of_day.strftime("%Y/%m/%d %H:%M")
+    @files = Assignment.new
```

Comment:
But they're all instances of Assignment -- how could they have different properties?  Esp. since you're only creating a single assignment right now, how does this help?